app/controllers/idv/verify_info_controller.rb

```diff
@@ -34,15 +34,25 @@ def show
       process_async_state(load_async_state)
     end
 
+    def update
+      success = shared_update
+
+      if success
+        # Don't allow the user to go back to document capture after verifying
+        if flow_session['redo_document_capture']
```

Comment:
See [LG-9996](https://cm-jira.usa.gov/browse/LG-9996) and [slack thread](https://gsa-tts.slack.com/archives/C03DCUECFH7/p1686158394565189). VerifyInfo shows the user the option to redo document capture if they had a barcode read error. The [code that lets the user redo](https://github.com/18F/identity-idp/blob/736c5fc877c95dbf6d7dfd8612ca51a80aa86a35/app/views/idv/verify_info/show.html.erb#L21-L36) is in the in_person verify_info show template, but I think it can be removed, since people can't get to the in person flow unless their ID couldn't be scanned.

Reply:
Yeah, I agree that that code can be deleted from the in_person verify_info show template. The `      @had_barcode_read_failure` is not set in our controller. I can open a separate ticket for that work. 

Thanks for flagging this!